### PR TITLE
Creative Commons Global Summit 2015 Documentation

### PR DESCRIPTION
**Pull request by ticoniq**
### **Creative Commons Global Summit 2015 Documentation**

This pull request updates the documentation for the Creative Commons Global Summit 2015. The changes include:

- Set up the repository and tools.
- Adapt the 2 pages to the desktop and mobile view.
- Implement the section "Featured speakers" with dynamic HTML.
- Added speaker bios and session descriptions
- Updated schedule with accurate timing information
- Improved readability and organization of content

These updates aim to provide accurate and comprehensive information about the summit, making it easier for attendees and future participants to access valuable resources and understand the event's agenda.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  <!-- Footer -->
+  <footer class="footer">
+    <div class="container grid">
+      <img src="assets/images/cck-logo1.png" alt="creative common korea">
+      <div class="div">
+        <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+      <p>달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다. 자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+      </div>
+    </div>
+  </footer>
```
Comment (ShadyShawkat) [inline index.html:167]: - [ ] Please be sure to remove the "Partners" and "footer" sections from the home page on the mobile version to match the given design.

image

File: README.md [hunk 2]
```diff
@@ -1 +1,193 @@
...
+- **Contact form for easy communication**
+- **Information about my work experience and educational background**
+- **Detailed project descriptions with links to code repositories and live demos**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Live Demo -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+<a href="https://ticoniq.github.io/Ist-Capstone-Project"> See Project 🚀</a>
```
Comment (ShadyShawkat) [inline README.md:69]: - [ ] Kindly make sure that the live demo link in the readme is working properly. Currently, this is what shows when the link is clicked.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @ticoniq, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ticoniq) [reply README.md:69]: @ShadyShawkat it's showing like that because I haven't merged the branch

**Pull request merged**